src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-units/attachment-units.component.ts

```diff
@@ -0,0 +1,181 @@
+                    this.alertService.error(res.error.title);
+                } else {
+                    onError(this.alertService, res);
+                }
+                this.isLoading = false;
+            },
+        });
+    }
+
+    cancelSplit() {
+        this.router.navigate(['course-management', this.courseId.toString(), 'lectures', this.lectureId.toString()]);
+    }
+
+    addRow() {
+        const unitDynamic = {
+            unitName: '',
+            startPage: 0,
+            endPage: 0,
+        };
+        this.units.push(unitDynamic);
+        return true;
+    }
+
+    deleteRow(i: number) {
+        if (this.units.length === 1) {
+            return false;
+        } else {
+            this.units.splice(i, 1);
+            return true;
+        }
+    }
+
+    validUnitInformation(): boolean {
+        for (const unit of this.units) {
+            if (unit.unitName === '' || unit.unitName === undefined) {
+                this.invalidUnitTableMessage = this.translateService.instant(`artemisApp.attachmentUnit.createAttachmentUnits.validation.empty.unitName`);
+                return false;
+            }
+
+            if (unit.startPage === undefined || unit.startPage === null) {
+                this.invalidUnitTableMessage = `${this.translateService.instant(`artemisApp.attachmentUnit.createAttachmentUnits.validation.empty.startPage`)}`;
+                return false;
+            }
+
+            if (unit.endPage === undefined || unit.endPage === null) {
+                this.invalidUnitTableMessage = `${this.translateService.instant(`artemisApp.attachmentUnit.createAttachmentUnits.validation.empty.endPage`)}`;
+                return false;
+            }
+
+            if (unit.startPage < 1) {
+                this.invalidUnitTableMessage = this.translateService.instant(`artemisApp.attachmentUnit.createAttachmentUnits.validation.startPage`);
+                return false;
+            }
+
+            if (unit.startPage > this.numberOfPages) {
+                this.invalidUnitTableMessage = `${this.translateService.instant(`artemisApp.attachmentUnit.createAttachmentUnits.validation.startPageBigger`)} ${
+                    this.numberOfPages
+                }`;
+                return false;
+            }
+
+            if (unit.endPage < 1) {
+                this.invalidUnitTableMessage = this.translateService.instant(`artemisApp.attachmentUnit.createAttachmentUnits.validation.endPageLower`);
+                return false;
+            }
+
+            if (unit.endPage > this.numberOfPages) {
+                this.invalidUnitTableMessage = `${this.translateService.instant(`artemisApp.attachmentUnit.createAttachmentUnits.validation.endPage`)} ${this.numberOfPages}`;
+                return false;
+            }
+        }
+
+        this.invalidUnitTableMessage = undefined;
+        return true;
+    }
+
+    get currentTimeZone(): string {
+        return Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
Why is this needed? This should also probably the time zone of the course?

Reply:
I display current time zone in attachment-units component how else can i access the time zone from here?
![timezone](https://user-images.githubusercontent.com/26312757/212131626-88057353-d2b4-43c0-a5bc-5fcb572e324e.png)

Reply:
I would just not show it here, it's also not shown in the other places where dates get input.